PWGCF/MultiparticleCorrelations/Core/MuPa-DataMembers.h

```diff
@@ -28,96 +31,159 @@
 
 // *) Task configuration:
-TString fTaskName = "";   // task name - this one is used to get the right weights programatically for this analysis
-Bool_t fVerbose = kFALSE; // print additional info like Green(__PRETTY_FUNCTION__); etc., to be used during debugging
-// TString fModusOperandi; // "Rec" = process only reconstructed, "Sim" = process only simulated, "RecSim" = process both reconstructed and simulated
-// UInt_t fRandomSeed; // argument to TRandom3 constructor. By default it is 0, use SetRandomSeed(...) to change it
-// Bool_t fUseFisherYates; // use SetUseFisherYates(kTRUE); in the steering macro to randomize particle indices
-// Bool_t fUseFixedNumberOfRandomlySelectedParticles; // use or not fixed number of randomly selected particles in each event. Use always in combination with SetUseFisherYates(kTRUE)
-// Int_t fFixedNumberOfRandomlySelectedParticles; // set here a fixed number of randomly selected particles in each event. Use always in combination with SetUseFisherYates(kTRUE)
-// Bool_t fRescaleWithTheoreticalInput; // if kTRUE, all measured correlators are rescaled with theoretical input, so that in profiles everything is at 1. Used both in OTF and internal val.
+TString fTaskName = ""; // task name - this one is used to get the right weights
+                        // programatically for this analysis
+TString fRunNumber = ""; // over which run number this task is executed
+Bool_t fVerbose =
+    kFALSE; // print additional info like Green(__PRETTY_FUNCTION__); etc., to
+            // be used during debugging, but not for function calls per particle
+Bool_t fVerboseForEachParticle =
+    kFALSE; // print additional info like Green(__PRETTY_FUNCTION__); etc., to
+            // be used during debugging, also for function calls per particle
+Bool_t fUseCCDB =
+    kFALSE; // access personal files from CCDB (kTRUE, this is set as default in
+            // Configurables), or from home dir in AliEn (kFALSE, use with care,
+            // as this is discouraged)
+// TString fModusOperandi; // "Rec" = process only reconstructed, "Sim" =
+// process only simulated, "RecSim" = process both reconstructed and simulated
+// UInt_t fRandomSeed; // argument to TRandom3 constructor. By default it is 0,
+// use SetRandomSeed(...) to change it Bool_t fUseFisherYates; // use
+// SetUseFisherYates(kTRUE); in the steering macro to randomize particle indices
+// Bool_t fUseFixedNumberOfRandomlySelectedTracks; // use or not fixed number of
+// randomly selected particles in each event. Use always in combination with
+// SetUseFisherYates(kTRUE) Int_t fFixedNumberOfRandomlySelectedTracks; // set
+// here a fixed number of randomly selected particles in each event. Use always
+// in combination with SetUseFisherYates(kTRUE) Bool_t
+// fRescaleWithTheoreticalInput; // if kTRUE, all measured correlators are
+// rescaled with theoretical input, so that in profiles everything is at 1. Used
+// both in OTF and internal val.
```

Comment:
Same comment regarding leaving commented code

Reply:
This code will be used in the future!